rasa/data.py

```diff
@@ -61,28 +85,28 @@ def get_core_nlu_directories(
         containing the NLU training files.
     """
 
-    story_files, nlu_data_files = get_core_nlu_files(paths)
+    story_files = get_data_files(paths, is_story_file)
+    nlu_data_files = get_data_files(paths, is_nlu_file)
 
     story_directory = _copy_files_to_new_dir(story_files)
     nlu_directory = _copy_files_to_new_dir(nlu_data_files)
 
     return story_directory, nlu_directory
 
 
-def get_core_nlu_files(
-    paths: Optional[Union[Text, List[Text]]]
-) -> Tuple[List[Text], List[Text]]:
+def get_data_files(
+    paths: Optional[Union[Text, List[Text]]], filter_property: Callable[[Text], bool]
+) -> List[Text]:
     """Recursively collects all training files from a list of paths.
 
     Args:
         paths: List of paths to training files or folders containing them.
 
     Returns:
-        Tuple of paths to story and NLU files.
+        paths of training data files.
     """
```

Comment:
please add the new argument inc. what the options are or where to find those options